learn/guides/asyncapi-tool.md

```diff
@@ -10,27 +10,27 @@ redirect_from:
   - /learn/ballerina-asyncapi-support
   - /learn/ballerina-asyncapi-support/
   - /learn/asyncapi-tool
---- 
+---
 
 Ballerina AsyncAPI tool will make it easy for you to start the development of an event API documented in an AsyncAPI contract in Ballerina by generating a Ballerina service and listener skeletons.
 
 > **Prerequisite:** Install the latest [Ballerina Swan Lake distribution](https://ballerina.io/downloads/).
 
-## Generate Ballerina services from AsyncAPI contracts
+## Generate Ballerina sources from AsyncAPI contracts
 
-If you prefer the design-first approach, use an existing or your AsyncAPI definition to generate Ballerina services using the AsyncAPI CLI command below.
+If you prefer the design-first approach, use an existing or your AsyncAPI definition to generate Ballerina sources using the AsyncAPI CLI command below.
 
 ```bash
 bal asyncapi -i <asyncapi-contract>
 ```
 
-The generated service can be used as a code template to start the service implementation. For example,
+The generated sources can be used as a code template to start the service implementation. For example,
```

Comment:
```suggestion
The generated sources can be used as code templates to start the service implementation. For example,
```

Reply:
Thanks a lot for the suggestions, Resolved all